beacon-chain/core/blocks/attestation.go

```diff
@@ -248,7 +249,7 @@ func VerifyIndexedAttestation(ctx context.Context, beaconState *stateTrie.Beacon
 	indices := indexedAtt.AttestingIndices
```

Comment:
Shouldn't those be already a slice of `types.ValidatorIndex`?

Reply:
Not currently, I have to open a TODO in fssz. It doesn't support HTR of `[]ValidatorIndex`, only `[]uint64`
Thanks for the reminder, Will open an issue now